mock/pubSubSubscriptionStub.go

```diff
@@ -0,0 +1,37 @@
+package mock
```

Comment:
Not needed for IsInterfaceNil ?

Reply:
not needed as the main implementation is the one from libp2p which does not have IsInterfaceNil